python/cuml/_thirdparty/sklearn/preprocessing/_column_transformer.py

```diff
@@ -30,6 +30,9 @@
 import cupy as np
 import numba
 
+import pandas
```

Comment:
can we do the usual `import pandas as pd` standard way, I think we do it that way in the rest of the codebase